content/en/agent/faq/ec2_imdsv2_transition_payload_enabled.md

```diff
@@ -0,0 +1,22 @@
+---
+title: IMDSv2 Enablement by Default
+aliases:
+  - /ec2_imdsv2_transition_payload_enabled
+---
+
+With the release of Agent v7.64.0, Datadog defaults to using IMDSv2 for enhanced security and consistency in retrieving AWS metadata.
+If your host has already configured the `ec2_prefer_imdsv2` flag to `true`, you won't notice any changes as IMDSv2 is already enabled.
+
+If you upgrade to Agent v7.64.0 without any custom configuration changes, you may see a change to how hostnames appear in Datadog. After this update, Datadog displays hostnames as instance IDs in Fleet Automation and on the Infrastructure list.
```

Comment:
```suggestion
If you upgrade to Agent v7.64.0 without any custom configuration changes, you may see a change to how hostnames appear in Datadog. In particular, displayed hostnames will strongly ensure that they refer to instance IDs rather than other detected hostnames in Fleet Automation and on the Infrastructure list.
```

Reply:
I don't think it's the hostnames themselves that perform the act of ensuring (pedantic, I know, but it's documentation)

Let's change that sentence: `s/In particular, displayed hostnames will strongly ensure that they refer to instance IDs rather than other detected hostnames in Fleet Automation and on the Infrastructure list./In particular, Datadog ensures that displayed hostnames refer to instance IDs rather than other detected hostnames in Fleet Automation and on the Infrastructure list.`